### PR TITLE
removes references to React Pagination component

### PR DESCRIPTION
**Pull request by rmessina1010**
## Chromatic

https://rm-remove-react-pagination-phase3--60f9b557105290003b387cd5.chromatic.com

## Description
Closes [2026](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2026)
Removes React Pagination component

## Screenshots
N/A

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "20.0.0",
+  "version": "22.0.0",
```
Comment (jamigibbs) [inline packages/core/package.json:4]: Is there a reason this needs to be incremented by two major releases?

File: packages/react-components/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/react-components",
-  "version": "15.0.0",
+  "version": "17.0.0",
```
Comment (jamigibbs) [inline packages/react-components/package.json:3]: Also bumped two major releases here.

**Review by jamigibbs — COMMENTED** [review]
Can you make sure to link to the actual Github issue in the description and not zenhub? That will automatically cross link for historical context. You can get the Github link here:

Screenshot 2023-09-11 at 9 38 34 AM

File: packages/core/package.json (hunk 1 above)
Comment (rmessina1010) [reply packages/core/package.json]: I made this change prior to our meeting on versioning. I have reverted this change to reflect the current version (20.0.0) for the appropriate engineer to update  core's version on release.

File: packages/react-components/package.json (hunk 2 above)
Comment (rmessina1010) [reply packages/react-components/package.json:3]: This  was set in anticipation. Andrew's deprecation of Select merged ahead of me @16.0.0, my deprecation of  Pagination, also a braking change , should make 17.0.0 correct

**Review by jamigibbs — APPROVED** [review]
🎉

**Pull request merged**